content/news/core-monthly/2023/core-monthly-2023-11-01-2023-11-30.md

```diff
@@ -0,0 +1,420 @@
+
+In the changes that are being made you can see a great deal that is related to the migration of the back office to Symfony. The SymfonyLayout project still requires a few things but the whole project is slowly approaching completion. You can see [the details in EPIC available on our GitHub](https://github.com/PrestaShop/PrestaShop/issues/32875).
+
+The other big project that is part of PrestaShop 9 is, of course, the implementation of [ApiPlatform](https://api-platform.com/) to create a full-fledged API for the back office. It's a big project, but the whole thing is taking shape and as those who had the opportunity to be at the PrestaShop developers conference already know - the feature will be available in PrestaShop 9 and thanks to its modularity it will be possible for the whole community to work on making the API coverage better, so that back office events like adding products, categories, changing configurations or managing discounts, will be implemented as soon as possible. If you want to know more about the API, I invite you to the next [Live Update](https://www.youtube.com/watch?v=7CQ2Sg-v4XQ). where we will talk about it in more detail.
+
+Another thing worth mentioning is updated jQuery, but not only in front office or back office, it has been updated everywhere thanks to work from [@Hlavtox](https://github.com/Hlavtox). You can see all the details in the [Upgrade jquery to latest version, drop polyfills PR](https://github.com/PrestaShop/PrestaShop/pull/34382/). This is a big change that definitely can impact your modules, so please check it out and make sure your modules are compatible with the new version of jQuery. 
+
+The new major versions also allow us to change, or fix, behaviors that don't quite work as they should and needed such changes. An example? Today, when we add a product to the shopping cart, and it is only available in quantity of 1 item, after adding it to the cart we will see a message on the product card that says that the product is no longer available. This could cause confusion for shoppers, sometimes they would leave the store thinking that someone was faster than them and made a purchase earlier. Starting with PrestaShop 9, the whole thing will work differently. You'll have an option to [Inform customer about quantity in cart](https://github.com/PrestaShop/PrestaShop/pull/33931).
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox), we also have some other interesting changes:
+- He is still working on a project to change the way product properties are fetched, so that instead of Product::getProductProperties we use lazy property loading thanks to ProductLazyArray. One part of this project is to [remove references to the `getProductProperties` method from where products are fetched](https://github.com/PrestaShop/PrestaShop/pull/34498). This should speed up all front office.
```

Comment:
```suggestion
- Work is underway on a project to change how product properties are fetched, so that instead of Product::getProductProperties we use lazy property loading thanks to ProductLazyArray. One part of this project is to [remove references to the `getProductProperties` method from where products are fetched](https://github.com/PrestaShop/PrestaShop/pull/34498). This should speed up all front office.
```